panic:Broken pipe (os error 32)

# tea does not run after installation

## Description

I tried to use tea on MacOS.
I got an error by removing `--silent` flag on tea.

## Environment

MacOS 13.2.1
tea 0.25.1

## Reproduction

`sh <(curl https://tea.xyz) && source <("$HOME/.tea/tea.xyz/v*/bin/tea" --magic=bash)`

## Result

```
$ sh <(curl https://tea.xyz) && source <("$HOME/.tea/tea.xyz/v*/bin/tea" --magic=bash)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 13557  100 13557    0     0   138k      0 --:--:-- --:--:-- --:--:--  142k

  hi 👋 let’s set up tea

  • we’ll put it here:  /Users/jwel/.tea
  • everything tea installs goes there
  • (we won’t touch anything else)

  │ docs https://github.com/teaxyz/cli#getting-started

grep: /Users/jwel/.bashrc: No such file or directory

  may we interest you in some magic?

  tea’s shell magic is optional but it’s the way it’s meant to be used.

  │ docs https://github.com/teaxyz/cli#magic

  Added:

   test -d "$HOME/.tea" && source <("$HOME/.tea/tea.xyz/v*/bin/tea" --magic=bash --silent)


  you’re all set!

  try it out:

   exec bash -i   # or open a new tab
   wget -qO- tea.xyz/white-paper | glow -

panic: spilt tea. we’re sorry and we’ll fix it… but you have to report the bug!

    https://github.com/teaxyz/cli/issues/new?title=panic%3ABroken+pipe+(os+error+32)

----------------------------------------------------->> attachment begin
BrokenPipe: Broken pipe (os error 32)
    at async file:///private/var/folders/8b/n_gmhmk57ks85swq3mrdbrwc0000gp/T/7a834af1/src/app.ts:111:13
<<----------------------------------------------------- attachment end
```

--- a/src/app.magic.ts
+++ b/src/app.magic.ts
@@ -99,7 +99,7 @@ export default function(self: Path, shell?: string) {
     return undent`
       cd() {
         builtin cd "$@" || return
-        source <("${d}"/tea +tea.xyz/magic -Esk --chaste env)
+        source /dev/stdin <<<"$("${d}"/tea +tea.xyz/magic -Esk --chaste env)"
       }
 
       if [[ "$PATH" != *"$HOME/.local/bin"* ]]; then

